wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/wsplugins/model/PluginBase.java

```diff
@@ -95,6 +99,23 @@ public void setEndpoints(List<ChePluginEndpoint> endpoints) {
     this.endpoints = endpoints;
   }
 
+  /** List of environment variables to set in the workspace */
```

Comment:
I think it would be clearer `variables to set in all machines of the workspace`

Reply:
I don't want to use the term `machine` here. Looks like we need to get rid of it because it is not clear in containers world. I changed the doc but used containers term instead.